server/src/main/java/org/elasticsearch/search/aggregations/support/ValuesSourceParserHelper.java

```diff
@@ -51,12 +51,18 @@ private ValuesSourceParserHelper() {} // utility class, no instantiation
         declareFields(objectParser, scriptable, formattable, false, ValueType.STRING);
     }
 
-    public static <T> void declareGeoFields(
+    public static <T> void declareGeoPointFields(
             AbstractObjectParser<? extends ValuesSourceAggregationBuilder<ValuesSource.GeoPoint, ?>, T> objectParser,
             boolean scriptable, boolean formattable) {
         declareFields(objectParser, scriptable, formattable, false, ValueType.GEOPOINT);
     }
 
+    public static <T> void declareGeoFields(
+        AbstractObjectParser<? extends ValuesSourceAggregationBuilder<ValuesSource.Geo, ?>, T> objectParser,
+        boolean scriptable, boolean formattable) {
+        declareFields(objectParser, scriptable, formattable, false, ValueType.GEO);
```

Comment:
I thought geo aggs weren't scriptable? Should we just set `scriptable` to false?

Reply:
hmm. to be honest this is one of those "keep things as they were" sort of changes. looks like it is dependent on which aggregation calls this, and each one passes whether it is scriptable or not. geo_centroid and geo_distance set scriptable -> true.